modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-test/src/testIntegration/java/com/liferay/portal/workflow/metrics/rest/resource/v1_0/test/InstanceResourceTest.java

```diff
@@ -235,7 +235,56 @@ public void testGetProcessInstancesPageWithSortString() throws Exception {
 			(entityField, instance1, instance2) -> {
 				String entityFieldName = entityField.getName();
 
-				if (StringUtil.equals("userName", entityFieldName)) {
+				if (StringUtil.equals("assigneeName", entityFieldName)) {
+					instance1.setAssignees(
+						() -> {
+							User user = UserTestUtil.addUser(
+								RandomTestUtil.randomString(
+									NumericStringRandomizerBumper.INSTANCE,
+									UniqueStringRandomizerBumper.INSTANCE),
+								LocaleUtil.getDefault(),
+								"aaa".concat(
+									StringUtil.toLowerCase(
+										RandomTestUtil.randomString())),
+								RandomTestUtil.randomString(),
+								new long[] {TestPropsValues.getGroupId()});
+
+							List<Assignee> assignees = Arrays.asList(
+								new Assignee() {
+									{
+										id = user.getUserId();
+										name = user.getFullName();
+									}
+								});
+
+							return (Assignee[])assignees.toArray();
+						});
```

Comment:
Hey @MariMalta and @seltong,

just instantiate an `Assignee[]` directly to avoid unnecessary transformations, (Array -> List -> Array).

Thx